test/testdata/googleapis_test/gapic/lang/common.yaml

```diff
@@ -1,5 +1,6 @@
 common:
   toolkit_path: test/fake-repos/toolkit
+  staging_repo_dir: '${REPOROOT}/test/testdata/test_output/final'
```

Comment:
See lines 5-7 for example. You can only see the `final` bit, and then the <lang>/<api> appended